scripts/lib/utils

```diff
@@ -8,7 +8,7 @@ readonly SHIPYARD_CONSUMERS=(admiral lighthouse subctl submariner submariner-ope
 readonly OPERATOR_CONSUMES=(submariner lighthouse)
 readonly SUBCTL_CONSUMES=(submariner submariner-operator cloud-prepare lighthouse)
 readonly PROJECTS_PROJECTS=(cloud-prepare lighthouse submariner)
-readonly INSTALLER_PROJECTS=(submariner-charts submariner-operator)
+readonly INSTALLER_PROJECTS=(submariner-operator)
 readonly RELEASED_PROJECTS=(subctl)
```

Comment:
Doesn’t `submariner-charts` need to be added to `RELEASED_PROJECTS`?

Reply:
yeah